hazelcast/src/main/java/com/hazelcast/internal/partition/impl/MigrationManager.java

```diff
@@ -967,7 +967,7 @@ private void assignCompletelyLostPartitions() {
         private void processNewPartitionState(PartitionReplica[][] newState) {
             int migrationCount = 0;
             // List of migration queues per-partition
-            List<Queue<MigrationInfo>> migrationQs = new ArrayList<>(newState.length);
+            List<Queue<MigrationInfo>> listOfPartitionMigrationQueues = new ArrayList<>(newState.length);
```

Comment:
maybe just partitionMigrationQueues? Plural form already suggests it's a collection